doc/logos/README.md

```diff
@@ -1,52 +1,61 @@
-# scikit-learn Branding
+# scikit-learn Brand Guidelines
+
+This section contains information around scikit-learn's brand standards and the use of scikit-learn assets. The purpose of these guidelines are to ensure the scikit-learn logo remains consistent and recognizable across all uses and communications. These guidelines also provide a common language for referring to the logos and their components.
 
-This section contains scikit-learn's branding standards.
 File types:
 - `PNG` is a higher-quality compression format; size of file is generally larger
 - `ICO` file format refers to an image file format that contains small size computer icon images
 - `SVG` Scalable Vector Graphics (SVG) are an XML-based markup language for describing two-dimensional based vector graphics. They can be created and edited with any text editor or with drawing software.
 
+[Brand Name](/doc/logos/README.md#brand-name) | [Color Palette](/doc/logos/README.md#color-palette) | [Typography](/doc/logos/README.md#typography) | [Logos](/doc/logos/README.md#logos)
+
 ---
+<br>
 
-# scikit-learn: Font, Colors & Logos
+## Brand Name
+The official name of the package is __scikit-learn__. Do not abbreviate or otherwise alter the name. Always spell ‘scikit’ with a lowercase ‘s’.
 
```

Comment:
```suggestion
## Brand Name
The official name of the package is __scikit-learn__. Do not abbreviate or otherwise alter the name. Always spell ‘scikit’ with a lowercase ‘s’.

## History of Logo
The scikit-learn logo is inspired by an analysis of the [iris dataset​](https://archive.ics.uci.edu/ml/datasets/iris​).  The two petal-like shapes of the logo represent clusters from an algorithm.  The colors blue and orange were chosen as they are complementary, distinguishable and allow for vision perception.
```

Reply:
For [this reason](https://www.meganstodel.com/posts/no-to-iris/) we should maybe remove any reference to the iris dataset. What do you think? Moreover it is not completely sure that it was the reference for the logo.



https://www.meganstodel.com/posts/no-to-iris/

Reply:
@francoisgoupil ok, removed ref to iris dataset.